docs/pages/versions/unversioned/sdk/filesystem-next.mdx

```diff
@@ -112,6 +112,41 @@ try {
 }
 ```
 
+### Uploading files using `@expo/fetch`
+
+You can upload files as blobs directly with fetch:
+
+```ts example.ts
+import { fetch } from 'expo/fetch';
```

Comment:
Is it `expo/fetch` or `@expo/fetch`? I was not able to find separate package in registry, so I guess it's part of `expo` package, and the header is incorrect.